### PR TITLE
feat(accordion): add unit tests for accordion (part3)

### PR DESCRIPTION
**Pull request by nadiia**

### CODE REVIEW

File: src/accordion/__tests__/panel.test.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    };
+    const event = {key: 'Enter'};
+    const wrapper = mount(<Panel {...props}>Content</Panel>);
+    const header = wrapper.find(StyledHeader);
+    header.simulate('keydown', event);
+    expect(props.onChange).toHaveBeenCalledWith({expanded: true});
+    expect(props.onKeyDown).toHaveBeenCalledWith(event);
+  });
+
+  test.only('onChange is called on a Space keydown event', () => {
```
Comment (gergelyke) [inline src/accordion/__tests__/panel.test.js]: I think you've accidentally left this `only` here

File: src/accordion/__tests__/panel.test.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    };
+    // $FlowFixMe
+    const wrapper = mount(<Panel {...props}>Content</Panel>);
+    const header = wrapper.find(StyledHeader);
+    header.simulate('click');
+    expect(props.onChange).toHaveBeenCalledWith({expanded: true});
+    expect(props.onClick).toHaveBeenCalled();
+  });
+
+  test('onChange is called on a Ender keydown event', () => {
```
Comment (gergelyke) [inline src/accordion/__tests__/panel.test.js]: typo, `Enter`?

File: src/accordion/__tests__/panel.test.js (hunk 1 above)
Comment (nadiia) [reply src/accordion/__tests__/panel.test.js]: good catch! removed

File: src/accordion/__tests__/panel.test.js (hunk 2 above)
Comment (nadiia) [reply src/accordion/__tests__/panel.test.js]: yep, fixed

**Pull request merged**